vidarr-server/src/main/resources/db/migration/V0019__delete_recovery.sql

```diff
@@ -0,0 +1,6 @@
+DELETE FROM active_operation WHERE workflow_run_id IN (SELECT active_workflow_run.id FROM active_workflow_run);
+DELETE FROM analysis_external_id WHERE analysis_external_id.external_id_id IN (SELECT external_id.id FROM external_id WHERE external_id.workflow_run_id IN (SELECT active_workflow_run.id FROM active_workflow_run));
+DELETE FROM external_id WHERE external_id.workflow_run_id IN (SELECT active_workflow_run.id FROM active_workflow_run);
```

Comment:
ERROR: update or delete on table "external_id" violates foreign key constraint "external_id_version_external_id_id_fkey" on table "external_id_version"

Reply:
Should be fixed.